src/components/PhoneInput/index.tsx

```diff
@@ -35,8 +34,7 @@ const StyledLabel = styled.label<PhoneInputLabelProps>`
   position: relative;
   display: flex;
   border-radius: 4px;
-  border: 1px solid
-    ${({ theme: { colorsDeprecated } }) => colorsDeprecated.gray350};
+  border: 1px solid ${({ theme }) => theme.colors.neutral.border};
```

Comment:
```suggestion
  border: 1px solid ${({ theme }) => theme.colors.neutral.borderWeak};
```

Same here background is white so we are in a weak context 😄

Reply:
done